### PR TITLE
Refine property assignment test 08

### LINKED ISSUE #79: Add IDE manual smoke test coverage for lazy property assignment
https://gradle.slack.com/archives/C02L5SDP3S7/p1689586186653739

We should add a smoke test for both DSLs

### PR DESCRIPTION
**Pull request by hegyibalint**
Expands upon the comments of #87.

Closes #79.

### CODE REVIEW

File: 08-property-assignment/build.gradle.kts [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+
+    @InputDirectory
+    val fileSystemLocationProperty = project
+        .objects
+        .directoryProperty()
+
+    @Input
+    val mapProperty = project
+        .objects
+        .mapProperty<Any, Any>()
```
Comment (asodja) [inline 08-property-assignment/build.gradle.kts:15]: You can probably just use `String` as key, it's well supported and recommended. Not sure how `Any` behaves

**Review by asodja — APPROVED** [review]
LGTM

File: 08-property-assignment/build.gradle.kts (hunk 1 above)
Comment (asodja) [reply 08-property-assignment/build.gradle.kts:15]: Also note: `ListProperty<X>` and `SetProperty<X>` have different assignment extensions compared to MapProperty. So having one of `ListProperty<X>` might also make sense. Although, the we already have maybe too many properties to check... 🤔

Comment (hegyibalint) [reply 08-property-assignment/build.gradle.kts:15]: Added

**Pull request merged**